scripts/demo/demo_benchmark_app.bat

```diff
@@ -95,7 +95,7 @@ set downloader_dir=%INTEL_OPENVINO_DIR%\deployment_tools\open_model_zoo\tools\do
 
 for /F "tokens=* usebackq" %%d in (
     `python "%downloader_dir%\info_dumper.py" --name "%model_name%" ^|
-        python -c "import sys, json; print(json.load(sys.stdin)[0]['subdirectory'])"`
+     python -c "import sys, json; print(json.load(sys.stdin)[0]['subdirectory'])"`
```

Comment:
This is a questionable change; this line is a continuation of the previous line, so it should be indented.

Reply:
Fixed. Thanks.